integration_test/test/code_generation/app_with_defaults_test.exs

```diff
@@ -161,4 +161,29 @@ defmodule Phoenix.Integration.CodeGeneration.AppWithDefaultsTest do
       end)
     end
   end
+
+  describe "phx.gen.auth" do
```

Comment:
What if we also add the name of the hashing lib in the describe? E.g.: phx.gen.auth+bcrypt